firebase-firestore/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 - [changed] Transactions now perform exponential backoff before retrying.
   This means transactions on highly contended documents are more likely to
   succeed.
+- [feature] Added a `terminate` method to `FirebaseFirestore` which can
```

Comment:
We usually include parens when referencing methods, i.e. "terminate()"...

Reply:
Done.